forc-util/src/lib.rs

```diff
@@ -409,10 +409,10 @@ fn construct_window<'a>(
     &input[calculated_start_ix..calculated_end_ix]
 }
 
-/// A subscriber built from default tracing_subscriber::fmt::SubscriberBuilder such that it would match directly using println! throughout the repo
-/// RUST_LOG environment variable can be used to set different minimum level for the subscriber, default is INFO
 const LOG_FILTER: &str = "RUST_LOG";
 
+/// A subscriber built from default tracing_subscriber::fmt::SubscriberBuilder such that it would match directly using println! throughout the repo
+/// RUST_LOG environment variable can be used to set different minimum level for the subscriber, default is INFO
```

Comment:
```suggestion
///
/// `RUST_LOG` environment variable can be used to set different minimum level for the subscriber, default is `INFO`.
```

Just a heads up that these two lines will be joined when the docs are rendered, so it would be worth adding a newline like this. Here's an example of how it looks now:

![Screenshot from 2022-05-20 17-29-56](https://user-images.githubusercontent.com/4587373/169476753-cbe50f12-65a2-45e7-a043-ee874ccbc6eb.png)

The extra newline inbetween will separate the two comments.

Reply:
Also in case you're not aware, you can test how the docs will render locally with `cargo doc --open`.